.github/workflows/buildAndTest.yml

```diff
@@ -175,7 +175,7 @@ jobs:
             -DCMAKE_MODULE_PATH=`pwd`/../cmake/modulesXilinx \
             -DMLIR_DIR=../llvm/install/lib/cmake/mlir \
             -DLLVM_DIR=../llvm/install/lib/cmake/llvm \
-            -DCMAKE_LINKER=lld \
+            -DLLVM_USE_LINKER=lld \
```

Comment:
[this flag sets](https://github.com/llvm/llvm-project/blob/4aae5387a874a55ee491f5dc23ce0506c5cdc678/llvm/cmake/modules/HandleLLVMOptions.cmake#L361).

```
-DCMAKE_EXE_LINKER_FLAGS_INIT="-fuse-ld=lld"
-DCMAKE_MODULE_LINKER_FLAGS_INIT="-fuse-ld=lld"
-DCMAKE_SHARED_LINKER_FLAGS_INIT="-fuse-ld=lld"
```